final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,61 @@
+import feedparser
+from pprint import pprint
+from myargparse import *
```

Comment:
Please, do not use `import *` in your solutions

Reply:
Ok